docs/API.md

```diff
@@ -0,0 +1,369 @@
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">cleanupAuction</td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">queryStandingBidState</td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">announceAuction</td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">discoverBidders</td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">authorizeBidders</td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">startBidding</td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">claimAuctionLotSeller</td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">enterAuction</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td colspan="2">+</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">discoverSellerSignature</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td colspan="2">+</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">placeBid</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td colspan="2">+</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">claimAuctionLotBidder</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">claimDepositLoser</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td align="left">announceDelegateGroup</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>+</td>
+    </tr>
+  </tbody>
+</table>
+
+
+## Operations available for all users
+
+### queryAuctions
+
+Query all currently existing auctions discoverable on-chain,
+filtering out invalid ones (i.e. having invalid auction terms).
+
+Takes no parameters.
+
+Returns the list of `AuctionInfo` values, which consists of `AuctionTerms` 
+plus some on-chain auction parameters, 
+like auction the token currency symbol and addresses of auction validators.
+
+### cleanupAuction
+
+The cleanup procedure is supposed to happen after the purchase deadline,
+burning down all auction-related assets and thus removing all information about the auction from the blockchain.
```

Comment:
I think we should also mention that during cleanup all auction-related leftover utxos are consumed, like auction escrow utxo and auction metadata utxo.